astro/src/content/quickstarts/quickstart-javascript-angular-web.mdx

```diff
@@ -0,0 +1,240 @@
+            lang="typescript"/>
+
+#### Create an Account Page
+
+The account page will be a simple page that displays a random balance for the logged in user. You can create a new component for this page with the Angular CLI:
+
+```shell
+npx ng generate component account-page
+```
+
+Then, add the balance to the `src/app/account-page/account-page.component.html` template:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-angular-web/main/complete-application/src/app/account-page/account-page.component.html"
+            lang="html"/>
+
+And the login function to the component `src/app/account-page/account-page.component.ts`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-angular-web/main/complete-application/src/app/account-page/account-page.component.ts"
+            lang="typescript"/>
+
+## Authentication
+
+You now have created a basic Angular application with a home page and an account page. The next step is to add authentication to the application.
+
+### Auth Guard
+
+The first step is to create an Auth Guard. This will be used to protect the account page and only allow logged in users to access it.
+
+Create a new file called `auth-guard.ts` in the `src/app` directory:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-angular-web/main/complete-application/src/app/auth-guard.ts"
+            lang="typescript"/>
+
+The Auth Guard is a functional router guard, which can be used to handle anonymous and logged in states. The first parameter defines if
+the guard should allow logged in users `true` or anonymous users `false`. The second parameter is the route to redirect to if the guard fails.
+
+### Routing
+
+Next, you'll need to add the Auth Guard to the routing configuration. Open the `src/app/app.module.ts` file and add the RouterModule with the following routes inside the `imports: [ ]` part:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-angular-web/main/complete-application/src/app/app.module.ts"
+            lang="typescript"
+            tags="routerConfiguration"/>
+
+Don't forget to import the `RouterModule` and `authGuard` at the top of the file.
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-angular-web/main/complete-application/src/app/app.module.ts"
+            lang="typescript"
+            tags="importAuthGuard"/>
+
+
+### Tie it all together
+
+Now that you have the Auth Guard and the routing configured, you can tie it all together. Open the `src/app/app.component.html` file and replace its content with the following:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-angular-web/main/complete-application/src/app/app.component.html"
```

Comment:
We are also including a functioning page for `make-change` link/route.

You can find an example of the logic in the Next.js example app.
https://github.com/FusionAuth/fusionauth-quickstart-javascript-nextjs-web/blob/main/complete-application/src/components/MakeChangeForm.tsx

Or a crude example in
https://github.com/FusionAuth/fusionauth-quickstart-javascript-express-web/tree/fix-make-change

Reply:
Requested some changes for make-change to match our other examples

https://github.com/FusionAuth/fusionauth-quickstart-javascript-angular-web/pull/2#pullrequestreview-1617650088